components/InterestZoneModal/UnsterilizedCat.tsx

```diff
@@ -52,7 +52,7 @@ const UnsterilizedCat = ({ cat, zoneID }: Props) => {
     setDisplayImage('/images/placeholder-cat.png');
     GetImages().then((resp) => {
       if (resp.length > 0) {
-        setDisplayImage(`data:image/png;base64,${resp[0]}`)
+        setDisplayImage(`data:image/png;base64,${resp[0].imageString}`)
```

Comment:
Please add typing to the `GetImages` method, and if not difficult to related methods as well